server/src/test/integration/org/apache/cassandra/sidecar/coordination/MostReplicatedKeyspaceTokenZeroElectorateMembershipIntegrationTest.java

```diff
@@ -107,12 +109,17 @@ void test(TestVersion version) throws IOException
 
     private void runTestScenario(AbstractCluster<?> cluster)
     {
-        List<MostReplicatedKeyspaceTokenZeroElectorateMembership> memberships = buildElectorateMembershipPerCassandraInstance(cluster);
+        var pair = buildElectorateMembershipPerCassandraInstance(cluster);
```

Comment:
using var here, not sure if this is something we should discourage

Reply:
can we avoid using `var`?

Reply:
I used it for convenience in the test, otherwise the line was huge! I'd prefer to keep it here, but I also am wary of introducing var because other contributors will start using it and I think we should lean on discouraging its use.

Reply:
ok, I'm no longer using var, I think that's better